modules/nf-core/ashlar/main.nf

```diff
@@ -0,0 +1,33 @@
+process ASHLAR {
+    tag '$meta.id'
+    label 'process_single'
+
+    conda "bioconda::ashlar=1.17.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ashlar:1.17.0--pyh5e36f6f_0' :
+        'quay.io/biocontainers/ashlar:1.17.0--pyh5e36f6f_0' }"
+
+    input:
+    tuple val(meta), path(file_in)
+
+    output:
+    tuple val(meta), path("*.tif")      ,   emit: ashlar_tif
```

Comment:
Maybe this should just be `emit: tif`. Reviewing a bunch of other modules, I almost never see the tool name used in output names. Also the pattern should really be `*.ome.tif`.

Reply:
Yes, you're right.  I've made the change and it passes the tests.